jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/DeployServerlessApplicationDialog.kt

```diff
@@ -24,9 +28,14 @@ import javax.swing.JComponent
 
 class DeployServerlessApplicationDialog(
     private val project: Project,
+    private val templateFile: VirtualFile,
```

Comment:
If you are passing in `templateFile`, the `parameters` parameter is redundant, since it will never be sourced from somewhere else.

Reply:
Good catch. I removed the parameter `parameters` here to derive directly from `templateFile`